config/opa-service/policy.rego

```diff
@@ -23,14 +23,20 @@ token = {"payload": payload, "header": header} {
 }
 
 # Valid Issuers
-valid_iss = {"http://localhost:8085/auth/realms/master"}
+valid_iss = "http://localhost:8085/auth/realms/master"
+
+# API URI
+api_uri = "http://policy-api:8000/v1/policies/"
+
+# Audience
+aud = "client1"
```

Comment:
we should move this values outside the "rule", otherwise the rule is not more generic

either we "store" this as configuration in opa data,
or we fetch them as env variables using https://www.openpolicyagent.org/docs/latest/policy-reference/#opa